corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -744,12 +749,14 @@ function GeoPointEntry(question, options) {
 
     self.search = function (form) {
         var query = $(form).find('.query').val();
-        self.geocoder.geocode({
-            'address': query,
-        }, function (results, status) {
-            if (status === google.maps.GeocoderStatus.OK) {
-                self.map.fitBounds(results[0].geometry.viewport);
-                self.map.setCenter(results[0].geometry.location);
+        self.geocoder.query(query, function(err, data) {
+            if (err != null) {
+                console.error(err);
```

Comment:
Do you know what kind of situations would lead to an error here? Should the error be visible to the user?

Reply:
I think it's just a generic error for cases like if the mapbox servers are not reachable. Yeah I don't know our convention on this, I saw in sentry we log to a `/js_error` path?

Reply:
For a generic error, the ideal thing would be to let the user know something has gone wrong. It looks like questions have an error observable; what happens if you do something like `question.error(gettext('Could not load map. Please try again later.'))`?